translations/combat.tex/es.po

```diff
@@ -359,19 +295,16 @@ msgstr ""
 
 #. type: multicols
 #: sections/combat.tex:103
-#, fuzzy, no-wrap
-#| msgid ""
-#| "{\\centering\\phantom{Star Axis}\\par}\n"
-#| "\\includegraphics[width=\\linewidth]{\\art/unicorn.jpg}\n"
-#| "\n"
+#, no-wrap
 msgid ""
 "\\begin{center}\n"
 "  \\includegraphics[width=0.55\\linewidth]{\\art/wyvern.png}\n"
 "\\end{center}\n"
 "\n"
 msgstr ""
-"{\\centering\\phantom{Eje Estelar}\\par}\n"
-"\\includegraphics[width=\\linewidth]{\\art/unicorn.jpg}\n"
+"\\begin{center}\n"
+"  \\includegraphics[width=0.55\\linewidth]{\\art/wyvern.png}\n"
+"\\end{center}\n"
 "\n"
```

Comment:
```suggestion
```
If there's nothing to translate, there's no need to write anything. Original string will be used.